src/main/java/daisy/error/IllegalDeadlineFormatException.java

```diff
@@ -0,0 +1,4 @@
+package daisy.error;
+
+public class IllegalDeadlineFormatException extends Exception{
```

Comment:
Can create specific methods in the exception that print specific error messages, exceptions act just like classes